bundles/org.openhab.binding.miio/src/main/resources/OH-INF/i18n/basic.properties

```diff
@@ -1548,7 +1565,7 @@ ch.zhimi.airp.mb4a-miot.filter_life_level = Filter - Filter Life Level
 ch.zhimi.airp.mb4a-miot.filter_used_time = Filter - Filter Used Time
 ch.zhimi.airp.mb4a-miot.miio_lib_version = Custom Service - Miio Lib Version
 ch.zhimi.airp.mb4a-miot.mode = Mode
-ch.zhimi.airp.mb4a-miot.moto_speed_rpm = Custom Service - Moto Speed Rpm
+ch.zhimi.airp.mb4a-miot.moto_speed_rpm = Custom Service - Motor Speed Rpm
```

Comment:
moto

Reply:
Yes, I left that intentionality as this was an existing device, so to avoid breaking channel changes.
So I kept the chants to the visual ones.

I can change if you think it is confusing, my thought was to minimize required user action for such a minimal difference

Reply:
OK